cmd/cli/kubectl-kyverno/commands/create/templates/aggregated-role.yaml

```diff
@@ -0,0 +1,26 @@
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  name: kyverno-{{.ResourceType}}-permission
+  labels:
+    rbac.authorization.k8s.io/aggregate-to-kyverno: "true"
+rules:
+  - apiGroups: ["{{.ApiGroup}}"]
+    resources: ["{{.ResourceType}}"]
+    verbs: [{{range .Verbs}}"{{.}}" ,{{end}}]
+
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRoleBinding
+metadata:
+  name: {{.ResourceType}}-binding
+subjects:
+{{- range .Controllers }}
+- kind: ServiceAccount
```

Comment:
Why do we need a Service Account?

The command should only generate the aggregated role for the specified Kyverno controllers. The permissions will then be merged to the existing ClusterRole which is already mapped to an SA.